src/models/decisions/classification.ts

```diff
@@ -0,0 +1,31 @@
+import { EnumValue } from "../../helpers/advancedTypes";
+import { Tag } from "../subject";
+import { IDecision, VerificationDecision } from "./decision";
+
+export interface IClassification {
+    tag: Tag;
+    confirmed: EnumValue<VerificationDecision>;
+    colorIndex: number;
+    identifier?: symbol;
+}
```

Comment:
redundant interface?

Reply:
Removed interface and changed constructor to use constructor argument injection instead of the anonymous object pattern